### ISSUE TITLE
Urgent GSoC revisions are needed.

### ISSUE THREAD

**Issue by henrykironde** [arviz-devs/arviz#1987]
https://github.com/numfocus/gsoc/issues/396
Google is not happy with our idea lists.
 
 
1. Be sure to include whether the project is a 175 hour (medium sized) or 350 hour (large project). If it can be either, please state that-per idea. 
`2. Your Ideas Page URL should be accessible to all and not require login. --- (Covered)`
3. As we state in the Defining a Project Idea List section of the Mentor guide, please provide the following information for each idea:
a) a project title/description 
b) more detailed description of the project (2-5+ sentences) 
c) expected outcomes 
d) skills required/preferred 
e) possible mentors 
f) expected size of project (175 or 350 hour) 
g) an easy, medium or hard difficulty rating of each project.
 
 
If you want to be selected this year I strongly suggest you make sure your Project Ideas list meets the above requirements ASAP.

**Comment by OriolAbril** [comment]
I will update the list now. Do you have any extra info on this?

> an easy, medium or hard difficulty rating of each project.

It might be only me but this makes no sense. Should we assume no familiarity with ArviZ but familiarity with the requirements listed? No familiarity with ArviZ nor with the requirements listed?

**Comment by OriolAbril** [comment]
Just updated https://github.com/arviz-devs/arviz/wiki/GSoC-2022-projects#arviz-dashboards-python to what I think matches the requirements above. Does that look good?

**Comment by henrykironde** [comment]
> It might be only me but this makes no sense. Should we assume no familiarity with ArviZ but familiarity with the requirements listed? No familiarity with ArviZ nor with the requirements listed?

I think it depends on a domain. I am assuming that ArviZ ideas are not research intensive. They are more of python programing and R. In that case medium works fine.

> Just updated https://github.com/arviz-devs/arviz/wiki/GSoC-2022-projects#arviz-dashboards-python to what I think matches the requirements above. Does that look good?

Seems like there are more some ideas with just one sentence in the description. This should apply to all ideas.

**Comment by OriolAbril** [comment]
> Seems like there are more some ideas with just one sentence in the description. This should apply to all ideas.

Yes, I am updating the other ideas now, just wanted to check the format was all right before going through all of them. Thanks

**Comment by henrykironde** [comment]
Seems tight to me, and thank you for the updates

**Comment by OriolAbril** [comment]
Just updated the rest of ideas. Let us know if more changes are needed.
